packages/react/src/utils/use-module-mutation-client.ts

```diff
@@ -0,0 +1,163 @@
+import { SigningCosmWasmClient } from '@cosmjs/cosmwasm-stargate'
+import { useAccount, useCosmWasmSigningClient, useRecentChains } from 'graz'
```

Comment:
This ties the consumers to use graz... while the most commonplace provider is cosmos-kit. Perhaps we should pull out the hooks that we would use into our own react hooks?

Reply:
Unless we also provide the graz context underneath...

Reply:
fixed